grammar.js

```diff
@@ -1033,9 +1031,9 @@ module.exports = grammar({
 
     // precedence is used to prevent conflict with assignment expression
     keyword_argument: $ => prec(1, seq(
-      $.identifier,
+      field("left",$.identifier),
       '=',
```

Comment:
I checked the tree-sitter-python repository and they use "name" and "value" instead of "left" and "right" for keyword args. It would probably be best to follow conventions defined in the primary tree-sitter repos here as well.

Reply:
Fixed